Jenkinsfile.upstream-operator-sync

```diff
@@ -0,0 +1,168 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+boolean goModModified = false
+operatorTagVersion = "upstream-sync"
+
+pipeline {
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+    // Needed for local build
+    tools {
+        jdk 'kie-jdk11'
+    }
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
```

Comment:
this is added automatically as part of `KogitoJobTemplate.createPipelineJob`, directly into the job definition, so can be removed